frontend-react/src/store/selectors/ui.js

```diff
@@ -0,0 +1,8 @@
+import { createSelector } from 'reselect';
+
+const uiSelector = state => state.ui;
+
+export const getDirectionSelector = createSelector(
+  uiSelector,
```

Comment:
This is the usual way I define selectors and imho it improves readability, especially if you define more selectors for a given part of the store. This may look redundant at this point, but for the other files with selectors this may be better, and then all selectors files will be consistent.

Reply:
Okay, makes sense. Thanks, i'll resolve it.